### PR TITLE
[Test] length of dims

### PR DESCRIPTION
**Pull request by jaeyun-jung**
1. Get the length of dimension, not constant.
2. Init dim as 0 in some testcase.

### CODE REVIEW

File: tests/test_models/models/custom_decoder.py [hunk 1]
```diff
@@ -71,7 +71,7 @@ def decode(self, raw_data, in_info, rate_n, rate_d):
                     fbb.String("")
                     fbb.Int(ttype)
                     with fbb.TypedVector():
-                        for j in range(8):
+                        for j in range(len(dims)):
```
Comment (wooksong) [inline tests/test_models/models/custom_decoder.py:74]:
```suggestion
                        for dim in dims:
                            fbb.Int(dim)
```

File: tests/test_models/models/custom_converter.py [hunk 2]
```diff
@@ -66,7 +66,7 @@ def convert(self, input_array):
             ttype = convert_to_numpy_type(tensor[1].AsInt)
             tdim = tensor[2].AsTypedVector
             dim = []
-            for j in range(4):
+            for j in range(len(tdim)):
```
Comment (wooksong) [inline tests/test_models/models/custom_converter.py:69]:
```suggestion
            for t in tdim:
                dim.append(t.AsInt)
```

File: tools/development/nnstreamerCodeGenCustomFilter.py [hunk 3]
```diff
@@ -121,13 +121,14 @@ def get_input(text):
...
 
   /**
    * Check max dimension (NNS_TENSOR_RANK_LIMIT) and
-   * fill 1 to uninitialized dimension values.
+   * fill 0 to uninitialized dimension values.
    */
-  for (i = 3; i < NNS_TENSOR_RANK_LIMIT; i++)
-    in_info->info[0].dimension[i] = 1;
+  for (i = 4; i < NNS_TENSOR_RANK_LIMIT; i++)
+    in_info->info[0].dimension[i] = 0;
```
Comment (niley7464) [inline tools/development/nnstreamerCodeGenCustomFilter.py:132]: Is there any different between nnstreamerCodeGenCustomFilter.py 's `getInputDim` and cppfilter_test.cc's `getInputDim`?
It's just a question cause there is 0 init only in nnstreamerCodeGenCustomFilter.py

File: tests/test_models/models/custom_converter.py (hunk 2 above)
Comment (jaeyun-jung) [reply tests/test_models/models/custom_converter.py]: updated, thanks.

File: tools/development/nnstreamerCodeGenCustomFilter.py (hunk 3 above)
Comment (jaeyun-jung) [reply tools/development/nnstreamerCodeGenCustomFilter.py:132]: Before calling the callback getinputdim(), tensor-filter initializes tensors-info struct. This 0-init code is just for clearly defining the tensor dimension.

File: tests/test_models/models/custom_decoder.py (hunk 1 above)
Comment (jaeyun-jung) [reply tests/test_models/models/custom_decoder.py]: also this is updated.

File: tests/nnstreamer_example/custom_example_passthrough/nnstreamer_customfilter_example_passthrough.c [hunk 4]
```diff
@@ -56,10 +52,6 @@ pt_init (const GstTensorFilterProperties * prop)
   data->info.dimension[1] = D2;
   data->info.dimension[2] = D3;
   data->info.dimension[3] = D4;
-  data->info.dimension[4] = D5;
-  data->info.dimension[5] = D6;
-  data->info.dimension[6] = D7;
-  data->info.dimension[7] = D8;
```
Comment (myungjoo) [inline tests/nnstreamer_example/custom_example_passthrough/nnstreamer_customfilter_example_passthrough.c:62]: Ok. it's initialized by 0. So, it's safe here.

File: tests/cpp_methods/cppfilter_test.cc [hunk 5]
```diff
@@ -180,8 +176,6 @@ filter_basic2::getOutputDim (GstTensorsInfo *info)
   info->info[0].dimension[1] = 16;
   info->info[0].dimension[2] = 16;
   info->info[0].dimension[3] = 2;
-  for (int i = 4; i < NNS_TENSOR_RANK_LIMIT; i++)
-    info->info[0].dimension[i] = 1;
```
Comment (myungjoo) [inline tests/cpp_methods/cppfilter_test.cc:184]: You cannot sure if info is properly initialized.

File: tests/nnstreamer_example/custom_example_LSTM/dummy_LSTM.c [hunk 6]
```diff
@@ -53,8 +52,7 @@ pt_init (const GstTensorFilterProperties * prop)
   data->info[0].dimension[0] = TSIZE;
   data->info[0].dimension[1] = TSIZE;
   data->info[0].dimension[2] = TSIZE;
-  for (i = 3; i < NNS_TENSOR_RANK_LIMIT; i++)
-    data->info[0].dimension[i] = 1;
```
Comment (myungjoo) [inline tests/nnstreamer_example/custom_example_LSTM/dummy_LSTM.c:57]: It's safe to do so here, though..

**Review by myungjoo — COMMENTED** [review]
You have some places ok to skip initializing. But, there are places that don't look ok to do so (in cppfilter_test.cc)

File: tests/cpp_methods/cppfilter_test.cc (hunk 5 above)
Comment (jaeyun-jung) [reply tests/cpp_methods/cppfilter_test.cc:184]: I added init code to get in/out dims. plz review the changes.

**Review by myungjoo — APPROVED** [review]

**Pull request merged**